src/Apps/Order/Components/Helper.tsx

```diff
@@ -0,0 +1,45 @@
+import { Sans } from "@artsy/palette"
+import React from "react"
+import { Spacer } from "Styleguide/Elements/Spacer"
+
+import { WithMediator } from "Router/WithMediator"
+import styled from "styled-components"
+
+const Link = styled.a`
+  text-decoration: underline;
+`
+
+interface HelperProps {
+  artworkId: string | null
+}
+
+export const Helper: React.SFC<HelperProps> = ({ artworkId }) => (
+  <WithMediator>
+    {mediator => (
+      <>
+        <Sans size="2" color="black60">
+          Have a question?{" "}
+          <Link
+            onClick={() => {
+              mediator && mediator.trigger("openOrdersBuyerFAQModal")
```

Comment:
Can `WithMediator` supply default props so that guards are no longer necessary? 

(One thing to note is that we will likely be migrating away from `<Subscribe>` soon and instead update ArtsyContext (located in `Components/Artsty`) to support the renderProps pattern; at that point `WithMediator` will become unnecessary, and `<Subscribe>`, if necessary, would be used exclusively for shared state changes over time involving components spread across a tree -- i.e., clicking a login button three levels down then updates a status in some header, etc.)

Reply:
> Can WithMediator supply default props so that guards are no longer necessary?

Good idea. Done :+1: